tests/src/actionContainers/Swift3ActionContainerTests.scala

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
If you run exactly the same tests as for `whisk/swiftaction`, maybe factor them out? Alternatively: add tests that exercise the swift3 features?